modules/packages/FFTW.chpl

```diff
@@ -600,8 +600,8 @@ module FFTW {
   //
   private proc checkRealCplxDimMismatch(realDom, complexDom, fnname, cplxarrdesc="") {
     const dim = realDom.rank;
```

Comment:
I think it'd be cleaner do `const dim = realDom.rank - 1` here and below.

Reply:
Where do you mean when you say "above"?

Reply:
Oh, maybe "below"?

Reply:
https://github.com/chapel-lang/chapel/pull/14522/commits/eae648bd42cb8000de2666ed2157470cbf2fb7a7 makes this change (if you want to make sure I got the two cases you were thinking of).